docs/modules/servers/pages/distributed/operate/cassandra-migration.adoc

```diff
@@ -17,13 +17,8 @@ Note that currently the progress can be tracked by logs.
 
 Here are the implemented migrations:
 
-* *From V5 to V6* : Goal is to no longer rely on an UDT partition key for mailboxPath tables. Entries will be migrated
-to mailboxPathV2 table relying on a composite primary key.
-* *From V6 to V7* : Goal is to populate mapping_sources projection table. This table allows finding the source of a given redirection, which is
-handy for things like mail aliases (I want to list aliases rewritting things to bob). Without this projection table being available,
-(ie we rely on schema version 6 or less) such information is obtained through a full table scan, unoptimized. From schema version 7,
-the optimized projection can safely be used.
-* *From V7 to V8* : Add UID_VALIDITY to mailboxPath table in order not to mandate mailbox table reads.
```

Comment:
Hmmm did we ever test versioning in our antora doc yet? That could be a good candidate to start :)

Reply:
:+1: but we only have the 3.6.0 tag yet

Reply:
but that change would be for the 3.7.0 right?

Reply:
Yes.